lib/web_of_science.rb

```diff
@@ -1,5 +1,5 @@
 # Web of Science (WOS) utilities
-module WOS
+module WebOfScience
```

Comment:
OK, if you insist, but it's intended to be a console utility and typing `WOS` is easier than `WebOfScience`.

Reply:
I suppose if your going to insist on this, then the rake task and namespace should also move from `wos` to `web_of_science` too.  So, either back out moving this WOS module or move the rake tasks too.

Reply:
I am not concerned with the rake task namespace in this PR (or probably ever).